jboss-kie-kieserver/added/launch/jboss-kie-kieserver.sh

```diff
@@ -719,3 +720,26 @@ function configure_server_state() {
 function generate_random_id() {
     cat /dev/urandom | env LC_CTYPE=C tr -dc 'a-zA-Z0-9' | fold -w 16 | head -n 1
 }
+
+function configure_jbpm_cluster(){
+    if [ ${KIE_JBPM_CLUSTER} = 'true' ]; then
+      configure_jbpm_cache
+      log_info "KIE_JBPM_CLUSTER enabled"
+    else
+      log_info "KIE_JBPM_CLUSTER disabled"
+    fi
+}
+
+function configure_jbpm_cache() {
+  local startTag='<cache-container name="server" aliases="singleton cluster" default-cache="default" module="org.wildfly.clustering.server">'
+  local jbpmTag="<cache-container name='jbpm'>\
```

Comment:
the eap c onfg files should use double quotes.

Reply:
ok

Reply:
ok